api/apps/api/test/scenario-run-results/world.ts

```diff
@@ -0,0 +1,114 @@
+import * as archiver from 'archiver';
+import * as unzipper from 'unzipper';
+import * as request from 'supertest';
+import { v4 } from 'uuid';
+import { createWriteStream, readFileSync, writeFileSync, unlinkSync } from 'fs';
+
+import { Repository } from 'typeorm';
+import { getRepositoryToken } from '@nestjs/typeorm';
+
+import { AppConfig } from '@marxan-api/utils/config.utils';
+import { MarxanExecutionMetadataGeoEntity } from '@marxan/marxan-output';
+import { DbConnections } from '@marxan-api/ormconfig.connections';
+
+import { bootstrapApplication } from '../utils/api-application';
+import { GivenUserIsLoggedIn } from '../steps/given-user-is-logged-in';
+import { GivenScenarioExists } from '../steps/given-scenario-exists';
+import { ScenariosTestUtils } from '../utils/scenarios.test.utils';
+import { GivenProjectExists } from '../steps/given-project';
+
+export const createWorld = async () => {
+  const app = await bootstrapApplication();
+  const token = await GivenUserIsLoggedIn(app);
+  const { projectId, cleanup } = await GivenProjectExists(app, token);
+  const scenario = await GivenScenarioExists(app, projectId, token);
+  const filesToRemove: string[] = [];
+
+  const marxanExecutionMetadataRepo: Repository<MarxanExecutionMetadataGeoEntity> = app.get(
+    getRepositoryToken(
+      MarxanExecutionMetadataGeoEntity,
+      DbConnections.geoprocessingDB,
+    ),
+  );
+
+  return {
+    cleanup: async () => {
+      filesToRemove.forEach((file) => unlinkSync(file));
+      await marxanExecutionMetadataRepo.delete({
+        scenarioId: scenario.id,
+      });
+      await ScenariosTestUtils.deleteScenario(app, token, scenario.id);
+      await cleanup();
+    },
+    GivenOutputZipIsAvailable: async () => {
+      const zipBuffer = await createArchive();
+      filesToRemove.push(...zipBuffer.filesToClean);
+      await marxanExecutionMetadataRepo.save(
+        marxanExecutionMetadataRepo.create({
+          outputZip: zipBuffer.archive,
+          inputZip: zipBuffer.archive,
```

Comment:
if output and input is the same, you can't recognize which one is really served

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/319/commits/e9e74a6875780002b457deec5bc20df6c0cae77e